### ISSUE TITLE
(Should) IndexedParameter values be articulately linked to name keys?

### ISSUE THREAD

**Issue by EliLazarus** [julia-mpsge/MPSGE.jl#120]
As of Indexed_Params_to_array_values, values for indexed parameters are entered as arrays. But that relies on an assumption that the indexing order and the parameter order are the same.
Probably the data should be some kind of named array (e.g. DenseAxisArray), and we should articulately enter the parameters with the specific index, right?

**Comment by mitchphillipson** [comment]
Yes, this is less of an issue now, but can still be annoying. We want JuMP like syntax to do

`@parameter(M, P[i=I], initial_p_values[i])`

This is one of post-documentation goals.

**Comment by EliLazarus** [comment]
I think this is done (woohoo!) and can be closed. 
The syntax issue is separate afaic.

**Comment by mitchphillipson** [comment]
This was implemented in #187
